code/modules/shuttle/gamemodes/crash.dm

```diff
@@ -46,6 +46,13 @@
 	. = ..()
 	SSshuttle.canterbury = src
 
+/obj/docking_port/mobile/crashmode/on_prearrival()
+	. = ..()
+	for(var/mob/living/carbon/human/human AS in GLOB.alive_human_list)
+		if(human.faction != FACTION_TERRAGOV)
+			return
+		human.play_screen_text("<span class='maptext' style=font-size:24pt;text-align:left valign='top'><u>[GLOB.operation_namepool[/datum/operation_namepool].get_random_name()]</u></span><br>" + "[SSmapping.configs[GROUND_MAP].map_name]<br>" + "[GAME_YEAR]-[time2text(world.realtime, "MM-DD")] █:█<br>" + "Black Ops Platoon<br>" + "[human.job.title], ████", /obj/screen/text/screen_text/command_order/intro/blackop)
```

Comment:
Was about to suggest that, thanks.
![image](https://user-images.githubusercontent.com/35091844/194703732-14a79934-7d65-4a99-86c7-42836d89e451.png)